source/_sponsors.erb

```diff
@@ -1,4 +1,4 @@
-<div id="sponsors">
+<aside class="sponsors">
   <% if has_sponsors? locals[:for_page] %>
     <h4>Meeting Sponsors</h4>
```

Comment:
![image](https://user-images.githubusercontent.com/325384/93710025-1c05b600-fb3b-11ea-99e5-2f66bd06bc68.png)

![image](https://user-images.githubusercontent.com/325384/93710034-2a53d200-fb3b-11ea-97ff-04fa290c5488.png)


The `<h4>` was kind of tricky on this one, but I decided an `<aside>` tag should make the semantics a bit more improved.

Reply:
It looks like `<h1>` is the right choice when in an aside https://www.w3.org/TR/2014/REC-html5-20141028/sections.html#the-aside-element :D